contracts/lib/LibPerpetuals.sol

```diff
@@ -1,6 +1,7 @@
 //SPDX-License-Identifier: GPL-3.0-or-later
 pragma solidity ^0.8.0;
 import "prb-math/contracts/PRBMathUD60x18.sol";
+import "hardhat/console.sol";
```

Comment:
This should be removed

Reply:
Oop

Reply:
https://hardhat.org/plugins/hardhat-log-remover.html